jib-core/src/main/java/com/google/cloud/tools/jib/hash/Digests.java

```diff
@@ -45,6 +45,10 @@ public static DescriptorDigest computeJsonDigest(List<? extends JsonTemplate> te
     return computeDigest(contents, ByteStreams.nullOutputStream()).getDigest();
   }
 
+  public static BlobDescriptor computeDigest(JsonTemplate template) throws IOException {
+    return computeDigest(template, ByteStreams.nullOutputStream());
+  }
+
   public static BlobDescriptor computeDigest(JsonTemplate template, OutputStream optionalOutStream)
```

Comment:
This `optionalOutStream` seems an odd name choice… since it's not optional.

Reply:
No more "optional"s.